CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
     * Update react-native to version 0.58+ and update App Center SDK packages.
     * Edit **app/build.gradle** to replace `compile` by `implementation`.
     * Run `react-native link` again.
+- Support a 256 character User ID string that applies to crashes, error, and push logs. Settable via `AppCenter.setUserId()`.
```

Comment:
Reuse the text from [Apple SDK changelog](https://github.com/Microsoft/AppCenter-SDK-Apple/blob/develop/CHANGELOG.md#appcenter-1):
> [Feature] Allow users to set userId that applies to crashes, error and push logs. This feature adds an API, but is not yet supported on the App Center backend.

Reply:
@dhei Thanks for pointing it out.

Reply:
Fixed the comment.